remo/reports/migrations/0019_activity_campaign_2015.py

```diff
@@ -14,15 +14,28 @@ def forwards(self, orm):
         to the Alumni group.
         """
         INACTIVATION_DATE = datetime.datetime(2015, 10, 1)
+        REGISTRATION_DATE = datetime.datetime(2015, 10, 4)
+
         User = orm['auth.User']
         alumni_group = orm['auth.Group'].objects.get(name='Alumni')
         inactive_users = (User.objects.filter(groups__name='Rep')
+                          .filter(userprofile__date_joined_program__lt=REGISTRATION_DATE,
+                                  userprofile__registration_complete=True)
                           .exclude(ng_reports__report_date__gt=INACTIVATION_DATE))
 
         for user in inactive_users:
             user.groups.clear()
             user.groups.add(alumni_group)
 
+            report = user.ng_reports.all().order_by('-report_date')[:1]
+            if report:
+                date_left_program = report[0].report_date
+            else:
+                date_left_program = datetime.datetime.now().date()
```

Comment:
This doesn't look like a good workaround when there is no last report (meaning no report). I suggest we add ``None`` instead.